nmrfx-utils/src/main/java/org/nmrfx/console/ConsoleController.java

```diff
@@ -107,14 +107,7 @@ public static ConsoleController create(InteractiveInterpreter interpreter, Strin
         consoleController = Fxml.load(ConsoleController.class, "ConsoleScene.fxml")
                 .withStage(stage)
                 .getController();
-
-        stage.show();
-        Screen screen = Screen.getPrimary();
-        Rectangle2D screenSize = screen.getBounds();
-        stage.toFront();
-        stage.setY(screenSize.getHeight() - stage.getHeight());
```

Comment:
I removed the code that calculates where to put the box initially because I would need to first show the stage and then hide it which will take time and also flash the console on the screen while starting up. This means that when the user clicks "Show Console" the first time it will pop up in the centre of the screen which I think should be acceptable because the user is actively looking for it, once they have moved the console where they want it, "Show Console" will pop it up where they put it. I can add this calculation back in if not acceptable